internal/kafka/internal/services/clusters_test.go

```diff
@@ -136,7 +136,7 @@ func Test_Cluster_Create(t *testing.T) {
 				cluster: mocks.BuildCluster(nil),
 			},
 			setupFn: func() {
-				mocket.Catcher.Reset().NewMock().WithQuery("INSERT").WithExecException()
+				mocket.Catcher.Reset().NewMock().WithQuery("INSERT").WithQueryException()
```

Comment:
Why this change?

Reply:
> Because with recent gorm changes the Save method performs INSERTs returning data. The same happens for the Create method

Does this mean that ‘WithExcecException’ no longer catches INSERT? It may be necessary to carefully review each test that utilizes `mocket`.

Reply:
The ones you see that have been changed in the PR are ones that were found because the tests started to fail with the old value

Reply:
Since the INSERT is no longer caught by the WithExecException, a test that was supposed to fail in case of an INSERT may now pass. 

This could result in the test missing a future change that introduces a bug that triggers the INSERT.

Reply:
Yep. Got that. We need to carefully review the tests to understand when it is the case and fix it. 
Even if the test is not failing, it doesn't mean it is working as intended, so just running them won't be enough.

Reply:
Hi @ziccardi it seems with the new gorm version recently released (1.25.0) they reverted that change and the `Save` method is no longer returning errors again.
I will update the PR again to not include that change and make the tests pass.

I think we can close the opened jira issue then.

Reply:
updated